test_data/yast/btrfs/btrfs+warnings_aarch64.yaml

```diff
@@ -0,0 +1,6 @@
+<<: !include test_data/yast/btrfs/common/btrfs+warnings_disks.yaml
+errors:
+  no_root: There is no device mounted at '/'
+warnings:
+  snapshots_small_root: Your root device is very small for snapshots
+  no_boot: Missing device for /boot/efi with size equal or bigger than 256 MiB and filesystem vfat
```

Comment:
checking your PR I understood a bit better the issue, so we have different message depending on the architecture. The cons I see is the fragmentation of test data when only one setting change and also that perhaps does not generalize the solution when we want to check any label text during testing with this framework (i.e.: final screen with the partitioning overview)

Other choice would be to duplicate the test you have with different name depending on the arch, no_boot_blablab and schedule conditionally or with different schedule files. The only line that would change would be the one getting the warning, which would be compared in a page containing all possible strings instead of test data. According to other discussions to duplicate the code we have for tests should not be an issue (but we can always share code between tests) and we would have all the warning in single place from the beginning, each one with some method to access it and each test calling that particular method. Also we would have only test data that is input or action of the user not something part of the application like labels. But maybe I'm not seeing the cons really well from my perspective.

Anyway, you don't need to change anything if you go for this solution, just sharing my thoughts about it and we will see how it goes :) I might have some opportunity to show the other way and I might discover that your choice was better and I will learn something.

Reply:
Thanks for the detailed explanation.

I just want to highlight, that this is not the last test module for warnings:) 

I guess there will be something around 7-8 test modules for them. After I'll add all of them, we'll be able to look on the overall picture and decide on how we can optimize and generalize the solution, if we will need so.